### PR TITLE
Tester rpc implementation

### PR DESCRIPTION
**Pull request by pasichDev**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
 project(
     NosoHive 
     VERSION 1.0.0 
     DESCRIPTION "NosoHive"
     HOMEPAGE_URL "https://github.com/NosoLab/NosoHive"
 )
 
+# Add libary CPR
+include(FetchContent)
+FetchContent_Declare(cpr GIT_REPOSITORY https://github.com/libcpr/cpr.git
```
[the remark this answers is not in the document]
Comment (pasichDev) [reply CMakeLists.txt:15]: This is the only working method supported by this library

**Review by pasichDev — COMMENTED** [review]
the code has passed the review

**Comment by cryppyrc** [comment]
code can be merged

File: tester-rpc/tester-rpc.cpp [hunk 2]
```diff
@@ -1,8 +1,112 @@
 ﻿#include <iostream>
+#include <string>
+#include <cpr/cpr.h>
+#include <nlohmann/json.hpp>
```
[the remark this answers is not in the document]
Comment (pasichDev) [reply tester-rpc/tester-rpc.cpp:5]: Of course, this is a better solution for readability.

**Pull request merged**